soundfile.py

```diff
@@ -935,19 +1110,21 @@ def read(self, frames=-1, dtype='float64', always_2d=False,
         """
         if out is None:
             frames = self._check_frames(frames, fill_value)
-            out = self._create_empty_array(frames, always_2d, dtype)
+            actual_out = self._create_empty_array(frames, always_2d, dtype)
```

Comment:
I'm a bit conflicted about this. It is one of Python's core features that we're dynamically typed. Here we pretend we aren't. To me, that reads like we're pretending we're not Python.

Reply:
Fixed in https://github.com/bastibe/python-soundfile/pull/462/commits/4b68fbf1f9e6f84c8a1d3d23129fe2f53cd8bb3d